.github/workflows/docker-node.yml

```diff
@@ -1,9 +1,14 @@
 name: Build Docker Images
 
 on:
-  push:
-    paths:
-      - '.github/docker-node/*.Dockerfile'
+  # TODO: revisit.
```

Comment:
Interesting find @ronilan!

Reply:
Issue opened: https://github.com/actions/runner/issues/1202 (and now I'm even more worried as to what triggers when)